docs/how-to-setup-freecodecamp-locally.md

```diff
@@ -474,23 +474,29 @@ If you are stuck, and need help, let us know by asking in the ['Contributors' ca
 There might be an error in the console of your browser or in Bash / Terminal / Command Line that will help identify the problem. Provide this error message in your problem description so others can more easily identify the issue and help you find a resolution.
 
 ### Troubleshooting
-If the app launches but you are encountering errors with the UI itself, for example if fonts are not being loaded or if the code editor is not displaying properly, you may try the following troubleshooting steps at least once:
+If the app launches but you are encountering errors with the UI itself, for example if fonts are not being loaded or if the code editor is not displaying properly, try the following if you are using docker:
```

Comment:
Just changing some of the phrasing as well as separating the different methods more clearly. This makes it slightly easier to follow along.

```suggestion
Upon launching the app, if you are encountering issues with the UI such as the fonts not loading or the code editor not displaying, try the following depending on your situation.

#### Using Docker
```

Reply:
This suggestion is not proper grammar and definitely does not feel like it makes it easier to read in my opinion.  It is fine starting with "If" and not fine to start with a prepositional phrase like "Upon launching the app".  How about something more like:
> If the app launches but you are encountering UI errors such as fonts not being loaded or the code editor not displaying properly, see the following depending on your local setup:

### Using Docker: